src/vs/base/browser/ui/codicons/codicon/codicon-animations.css

```diff
@@ -10,5 +10,6 @@
 }
 
 .codicon-animation-spin {
-	animation: codicon-spin 1.5s linear infinite;
+	/* Use steps to throttle FPS to reduce CPU usage */
+	animation: codicon-spin 1.5s steps(30) infinite;
```

Comment:
Note this is 30 steps per 1.5s, so 20fps (there might an off by 1 error in my calculation, not sure how steps work on repeat).

Reply:
Good point, thanks. I still think it's a good level of visual smoothness for the performance win here, even if that means it's 20FPS instead of 30FPS.